src/app/client/dashboard/trainwithcoach/page.jsx

```
@@ -0,0 +1,612 @@
+            className="text-red-500"
+            width={48}
+            height={48}
+          />
+          <p className="mt-4 text-lg font-medium text-white">
+            Failed to load trainers
+          </p>
+          <p className="mt-2 text-zinc-400">{error}</p>
+          <button
+            className="mt-4 rounded-lg bg-[#3E92CC] px-4 py-2 text-white hover:bg-[#2D7EB8] transition-colors"
+            onClick={() => window.location.reload()}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)